build-tools/heroku/tailwind/tailwind.full.js

```diff
@@ -1,350 +1,36 @@
+// Control the Tailwind file size by disabling stuff we don't need.
 module.exports = {
-  content: [],
-  presets: [],
-  darkMode: 'media', // or 'class'
+  content: [
+      '../../../templates/**/*.html',
+      '../../../main/**/*.md',
+      '../../../coursedata/**/*.md',
+      '../../../static/js/**/*.js',
+  ],
+  safelist: [
```

Comment:
Thanks for picking this up @TiBiBa! 

Question: why do we need to rewrite this whole file? Isn't it enough to add the `safelist` option to the existing config, and that's enough?

Also, wouldn't it make sense to commit the `generated.css` with all the options?

Reply:
(To take a step back maybe: do you agree with the premise of issue #2039? 😋)

Reply:
From some reason the `tailwind.full.js` file contains some changed to container width. So when generating the complete css file the layout was no longer consistent. I could figure out what caused it so copied the original config and added `safelist` to make sure all classes are generated.

Reply:
The changes between Tailwind v1 and v3 where pretty big, so we had to some manual tweaking when switching to v3 to make sure the UI stayed the same. However, this does not seem to work well with `tailwind.full.js`. So this seemed the most suitable solution. Not ideal, but not sure what would be better solution to still generate all classes.

Reply:
Right. I thought I regenerated using v3 but maybe I didn’t.

check in the generated.css though?

Reply:
Yes only the full generated config file from Tailwind v3 does not align with our UI. The color schemes, fonts and some paddings, margins etc. are different. So in the tweaked config file we reversed these changes by setting these settings to the values of the v1 config file. As these are not set in the full config file the layout is different when using that one to generate the css.

Reply:
👍